README.md

```diff
@@ -57,7 +61,6 @@ module "opensearch_domain" {
 
   domain_name     = "my-opensearch-domain"
   subnet_ids      = module.eks_cluster.private_subnet_ids
-  security_group_ids = module.eks_cluster.security_group_ids
```

Comment:
in the readme of the module itself you've kept it with these security groups.
Which one would you suggest to use?